client/my-sites/plan-features-2023-grid/index.jsx

```diff
@@ -503,78 +501,63 @@ export default connect(
 	( state, ownProps ) => {
 		const { placeholder, plans, isLandingPage, visiblePlans } = ownProps;
 
-		let planProperties = compact(
```

Comment:
A note that the changes below showed by GH are deceptively large, but it's just an indentation change upon removing `compact()`.